TV by the Numbers.js

```diff
@@ -1,144 +1,208 @@
 		}
+		item.complete();
+	});
 
-		// attach html
-		if (url == newItem.url)
-			newItem.attachments.push({title:PUB_TITLE+" Snapshot",
-				mimeType:"text/html", url:doc.location.href, snapshot:true});
-		
-		newItem.toString = function() { return this.title; };
-		items[newItem.url] = newItem;
-		post_count++;
+	translator.getTranslatorObject(function(trans) {
+		trans.itemType = "webpage";
+		trans.doWeb(doc, url);
+	});
+} /** BEGIN TEST CASES **/
+var testCases = [{
+		"type": "web",
+		"url": "http://tvbythenumbers.zap2it.com/page/20/?s=harry+potter",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "http://tvbythenumbers.zap2it.com/daily-ratings/monday-final-ratings-july-3-2017/",
+		"items": [{
+			"itemType": "webpage",
+			"title": "‘American Ninja Warrior’ adjusts up: Monday final ratings",
+			"creators": [{
+				"firstName": "Rick",
+				"lastName": "Porter",
+				"creatorType": "author"
+			}],
+			"date": "2017-07-06",
+			"abstractNote": "Final broadcast primetime live + same-day ratings for Monday, July 3, 2017 The top show on a rerun-filled Monday night saw its adults 18-49 rating grow from the preliminary numbers to the finals. &…",
+			"shortTitle": "‘American Ninja Warrior’ adjusts up",
+			"url": "http://tvbythenumbers.zap2it.com/daily-ratings/monday-final-ratings-july-3-2017/",
+			"websiteTitle": "TV By The Numbers by zap2it.com",
+			"attachments": [{
+				"title": "Snapshot"
+			}],
+			"tags": [
+				"american ninja warrior ratings",
+				"battle of the network stars ratings",
+				"daily ratings",
+				"kevin can wait ratings",
+				"life in pieces ratings",
+				"man with a plan ratings",
+				"mom ratings",
+				"scorpion ratings",
+				"so you think you can dance ratings",
+				"spartan: ultimate team challenge ratings",
+				"supergirl ratings",
+				"superhuman ratings",
+				"the bachelorette ratings",
+				"whose line is it anyway ratings"
+			],
+			"notes": [],
+			"seeAlso": []
+		}]
+	},
+	{
+		"type": "web",
+		"url": "http://tvbythenumbers.zap2it.com/sdsdskdh279882992z1/tv-ratings-friday-grimm-constantine-fall-shark-tank-the-amazing-race-last-man-standing-up-cristela-hawaii-five-0-steady/322517/",
+		"items": [{
+			"itemType": "webpage",
+			"title": "TV Ratings Friday: ‘Grimm’ & ‘Constantine’ Fall, ‘Shark Tank’, ‘The Amazing Race’ & ‘Last Man Standing’ Up, ‘Cristela’ & ‘Hawaii FIve-0’ Steady",
+			"creators": [{
+				"lastName": "TV By The Numbers",
+				"creatorType": "author",
+				"fieldMode": 1
+			}],
+			"date": "2014-11-01",
+			"abstractNote": "ABC was number one in adults 18-49 while CBS won with total viewers.",
+			"shortTitle": "TV Ratings Friday",
+			"url": "http://tvbythenumbers.zap2it.com/sdsdskdh279882992z1/tv-ratings-friday-grimm-constantine-fall-shark-tank-the-amazing-race-last-man-standing-up-cristela-hawaii-five-0-steady/",
+			"websiteTitle": "TV By The Numbers by zap2it.com",
+			"attachments": [{
+				"title": "Snapshot"
+			}],
+			"tags": [
+				"'",
```

Comment:
If the "Category" is blank (or a single quote) in all articles, it should be removed. Also title case works better than lower case for the tags. They're in title case within the article, if you want to pull those phrases instead of reworking what EM found